examples/plot_intersectional_bias.py

```diff
@@ -186,7 +186,7 @@
 # ~~~~~~~~~~
 #
 #   **"For a star, the location of the mean referred to the location of a singular real object,
```

Comment:
Should this be 'For a start' ?

Reply:
oops, thanks for catching that! fixed [f79d65e](https://github.com/fairlearn/fairlearn/pull/1378/commits/f79d65e250492a6cb70d6c0df7892defe7a7b7c3)